### PR TITLE
10.2.1 change log update

### PR DESCRIPTION
**Pull request by voroyam**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 2019-07-04
+
+10.2.1 update:
```
Comment (tboerger) [inline CHANGELOG.md]: Drop this line, that breaks the standard format.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+* Changed
+	* Upgrade oC version from 10.1.1 to 10.2.1 inc. SHA256
```
Comment (tboerger) [inline CHANGELOG.md]: `* Upgrade ownCloud version from 10.1.1 to 10.2.1` should be fine

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+	* Upgrade onlyoffice from 2.1.7 to 2.3.1 inc. SHA256
```
Comment (tboerger) [inline CHANGELOG.md]: `* Upgrade onlyoffice from 2.1.7 to 2.3.1` should be fine

File: CHANGELOG.md (hunk 1 above)
Comment (voroyam) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (voroyam) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 3 above)
Comment (voroyam) [reply CHANGELOG.md]: done

**Pull request merged**